src/software/networking/threaded_unix_sender.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <boost/asio.hpp>
+#include <boost/bind.hpp>
+#include <string>
+
+#include "software/networking/unix_sender.h"
+
+class ThreadedUnixSender
+{
+   public:
+    /**
+     * Creates a UnixSender that sends a string over the unix socket path.
+     *
+     * @param io_service The io_service to use to service outgoing SendProto data
+     * @param unix_socket_path The path to the unix socket
+     */
+    ThreadedUnixSender(const std::string& unix_socket_path);
```

Comment:
nit: there's only one argument in this function

Reply:
fixed